src/Consolonia.Themes/TurboVision/TurboVisionElegantColors.axaml

```diff
@@ -61,6 +61,9 @@
             <!--Just transparent background or foreground-->
```

Comment:
With Contrast theme buttons don't have background. I don't know if it is fine, just for your attention 
<img width="1056" height="756" alt="image" src="https://github.com/user-attachments/assets/78ddf36b-2140-42c3-873c-5d55904ce128" />